src/main/java/com/hivemq/cli/commands/swarm/AbstractSwarmCommand.java

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Instead of extending this class everythere needed you can use PicoCLI mixins.
```java
public class SwarmUrlOption {

    @CommandLine.Option(names = {"-url"}, defaultValue = "http://localhost:8080", description = "The URL of the HiveMQ Swarm REST API endpoint (default http://localhost:8888)", order = 1)
    @VisibleForTesting
    public @NotNull String commanderUrl = "http://localhost:8080";
}

And then in the currently extending classes use PicoCLI Mixins:
```java
   @CommandLine.Mixin
    private SwarmUrlOption swarmUrlOption = new SwarmUrlOption();
```
```